db/sql/schema_and_tables.sql

```diff
@@ -0,0 +1,56 @@
+--Schema
+
+CREATE SCHEMA IF NOT EXISTS facilities_lds;
+
+COMMENT ON SCHEMA facilities_lds IS
+'Schema that holds facilities data to publish on the LDS.';
+-- Tables
+
+-- nz facilities
+
+-- The nz_facilities table holds a multipolygon geometry, originating from National Map facility polygons and authoritative source data
+CREATE TABLE IF NOT EXISTS facilities_lds.nz_facilities (
+      facility_id serial PRIMARY KEY
+    , external_facility_id character varying(80) DEFAULT ''
+    , name character varying(250) DEFAULT ''
+    , external_name character varying(250) DEFAULT ''
+    , use character varying(40) NOT NULL DEFAULT ''
+    , use_type character varying(150) DEFAULT ''
+    , use_subtype character varying(150) DEFAULT ''
+    , estimated_occupancy integer DEFAULT 0
+    , last_modified date DEFAULT ('now'::text)::date
+    , internal boolean NOT NULL DEFAULT false
+    , internal_comments character varying(100) DEFAULT ''
+    , shape public.geometry(MultiPolygon, 2193) NOT NULL
+);
+
+CREATE INDEX shx_nz_facilities
+    ON facilities_lds.nz_facilities USING gist (shape);
+
+COMMENT ON TABLE facilities_lds.nz_facilities IS
+'The nz_facilities table holds geometries originating from NationalMap facility polygons '
+'and authoritative source data.';
+COMMENT ON COLUMN facilities_lds.nz_facilities.facility_id IS
+'The unique identifier for each geometry.';
+COMMENT ON COLUMN facilities_lds.nz_facilities.external_facility_id IS
+'The unique identifier of this facility used by the authoritative source';
+COMMENT ON COLUMN facilities_lds.nz_facilities.name IS
+'The name of the facility.';
+COMMENT ON COLUMN facilities_lds.nz_facilities.external_name IS
+'The name of the facility used by the authoritative source.';
+COMMENT ON COLUMN facilities_lds.nz_facilities.use IS
+'The generic use of the facility.';
```

Comment:
Do we want to mention the generic use matches those uses from Topo50?

Reply:
I wouldn't mention Topo50, as people might get confused and the link isn't that strong.  Also there are uses that are not in Topo50 like Supermarket and potentially Rest home.